ui/components/app/modals/customize-nonce/customize-nonce.component.js

```diff
@@ -52,9 +53,9 @@ const CustomizeNonce = ({
           >
             {t('editNonceField')}
           </Typography>
-          <button
-            className="fas fa-times customize-nonce-modal__close"
-            title={t('close')}
+          <ButtonIcon
+            iconName={ICON_NAMES.CLOSE}
+            className="customize-nonce-modal__close"
```

Comment:
Can some of these styles be removed in favour of Box props?

![Screenshot 2023-02-06 at 3 13 31 PM](https://user-images.githubusercontent.com/8112138/217108505-7e4525f7-b9ed-4db3-b92a-8d7c1bff8f8a.png)